iceberg/iceberg-handler/src/test/queries/positive/write_iceberg_branch.q

```diff
@@ -1,60 +0,0 @@
--- SORT_QUERY_RESULTS
```

Comment:
why did you drop this q test? should we move the testcases from `iceberg_branch_with_different_case.q` in here?